tempesta_fw/t/functional/tests/fragmented_requests.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+#A test for the Tempesta http parser with a request divided into chunks.
+# We get a request, divide it into chunks, then send the chunks separately.
+# Then check the Tempesta response.
+
+__author__ = 'Temesta Technologies Inc.'
+__copyright__ = 'Copyright (C) 2016 Tempesta Technologies Inc. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+import conf
+import tfw
+import socket
+import tfwparser
+import be
+import datetime
+class Test:
+	def fragmentize_str(self, s, frag_size):
+#		Split a string into a list of equal N-sized fragmen.
+#		>>> fragmentize_str("foo12bar34baz", 3)
+#		['foo', '12b', 'ar3', '4ba', 'z']
```

Comment:
Bad comment identation. I commented this many times before, i.e. https://github.com/tempesta-tech/tempesta/pull/462#discussion_r88301660